server/controllers/importers/sif.js

```diff
@@ -74,45 +74,52 @@ module.exports = function (sif) {
   });
 
   var genes = [];
-  entries.forEach(function (entry) {
-    if (entry.length && entry[GENE_NAME] && genes.indexOf(entry[GENE_NAME]) == constants.NOT_FOUND) {
-      genes.push(entry[GENE_NAME]);
+  var links = [];
+  var networkType = "unweighted";
+
+  if (entries.includes(null)) {
```

Comment:
`indexOf` may also work, but because `null` is special I would check on whether there are gotchas with it when used with `indexOf`.